### DISCUSSION TITLE
feat: shrink webview on keyboard opening (iOS)

### DISCUSSION THREAD

**Discussion by Aarbel** [ionic-team/capacitor#4917] [Q&A] [answered]
# Do capacitor supports webview shrink when the keyboard opens, especially for iOS ?

Cf cordova-plugin-keyboard: https://github.com/cjpearson/cordova-plugin-keyboard#cordovapluginskeyboardshrinkview

Do you plan to provide Keyboard.shrinkView() and Keyboard.disableScrollingInShrinkView() methods, are are they set by default ? 

Cf https://stackoverflow.com/questions/31179470/how-to-resize-web-view-when-keyboard-appears-on-ios-cordova

![image](https://user-images.githubusercontent.com/25119847/129073766-cb200eb9-c0ba-4b30-a0e4-3688c631a2da.png)

**Comment by jcesarmobile** [comment, marked as answer]
check [Keyboard plugin](https://capacitorjs.com/docs/apis/keyboard)

**Reply by Aarbel** [reply]
Ok great, it's supported thanks to https://capacitorjs.com/docs/apis/keyboard#keyboardresize, following this example: https://capacitorjs.com/docs/apis/keyboard#examples
